include/seastar/util/log.hh

```diff
@@ -214,20 +214,26 @@ public:
     /// \param args - args to print string
     ///
     template <typename... Args>
+#if FMT_VERSION >= 80000
+    void log(log_level level, fmt::format_string<Args...> fmt, Args&&... args) noexcept {
+#else
     void log(log_level level, format_info fmt, Args&&... args) noexcept {
+#endif
         if (is_enabled(level)) {
-            try {
-                lambda_log_writer writer([&] (internal::log_buf::inserter_iterator it) {
 #if FMT_VERSION >= 80000
-                    return fmt::format_to(it, fmt::runtime(fmt.format), std::forward<Args>(args)...);
+            lambda_log_writer writer([&] (internal::log_buf::inserter_iterator it) {
+                return fmt::format_to(it, fmt, std::forward<Args>(args)...);
+            });
```

Comment:
I feel like I'm missing totally obvious, but if `FMT_VERSION>=80000` the `do_log()` is not called and why is that change correct then?

Reply:
ahh, my bad. fixed and repushed.